ts/components/ui/ListItemTransaction.tsx

```diff
@@ -147,7 +142,9 @@ export const ListItemTransaction = ({
           <View
             style={{
               marginRight: IOListItemVisualParams.iconMargin,
-              marginLeft: IOListItemLogoMargin
+              marginLeft: IOListItemLogoMargin,
+              width: MUNICIPALITY_LOGO_SIZE,
+              alignItems: "center"
             }}
           >
```

Comment:
Looking this [Figma screen](https://www.figma.com/file/SuYjyjZlieQqRS8n1B6qbF/Flussi-App-ID-Pay?type=design&node-id=1017-21896&mode=design&t=eFKxUGca58sPMuXp-4), seems that `marginRight` should be 8px (_IOListItemLogoMargin_) and the marginRight was removed, so it should become as follow:

```tsx
<View
      style={{
        marginRight: IOListItemLogoMargin,
        width: PAYMENT_LOGO_SIZE,
        alignItems: "center"
      }}
    >
```

@dmnplb Please, can you check this behavior and confirm if it is correct?

Reply:
good catch! although in the design it is rendered in a padded container, it does look way better when removing the marginLeft.
addressed in [2308dea](https://github.com/pagopa/io-app/pull/4802/commits/2308dea8ae6acc155bab02ca75872d67082bb352)